PlayerControls/sources/DefaultControlsViewController.swift

```diff
@@ -227,16 +237,23 @@ public final class DefaultControlsViewController: ContentControlsViewController
             case (false, false):
                 setupBottomItemsConstraints()
                 setupBottomItems()
-                guard bottomItemsView.layer.animationKeys() == nil || state.isTransitioning else { return }
+                guard (bottomItemsView.layer.animationKeys() == nil &&
+                    seekerView.layer.animationKeys() == nil) ||
```

Comment:
Why we need to check for `animationKeys`?

Reply:
Because when the case `(true, false)` is trigger the animation of appearance starts. But because of playback, we get new props and we jump in `(false, false)` case. So if the `bottomItemsView` and `seekerView` have no animations, we can continue to do changes. 
The other thing there is `isTransitioning` variable. Because when transitioning starts, it also gives animation to all views, including `bottomItemsView` and `seekerView`. But in this case, these animations should be interupted and views have to jump on expected place. It was done to avoid flying views around the screen.